packages/grafana-ui/src/components/ColorPicker/NamedColorsPalette.story.tsx

```diff
@@ -1,21 +1,30 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
 import { NamedColorsPalette } from './NamedColorsPalette';
 import { getColorName, getColorDefinitionByName } from '@grafana/data';
 import { select } from '@storybook/addon-knobs';
 import { withCenteredStory } from '../../utils/storybook/withCenteredStory';
 import { renderComponentWithTheme } from '../../utils/storybook/withTheme';
 import { UseState } from '../../utils/storybook/UseState';
+import mdx from './ColorPicker.mdx';
 
 const BasicGreen = getColorDefinitionByName('green');
 const BasicRed = getColorDefinitionByName('red');
 const LightBlue = getColorDefinitionByName('light-blue');
 
-const NamedColorsPaletteStories = storiesOf('Pickers and Editors/ColorPicker/Palettes/NamedColorsPalette', module);
+// const NamedColorsPaletteStories = storiesOf('Pickers and Editors/ColorPicker/Palettes/NamedColorsPalette', module);
 
-NamedColorsPaletteStories.addDecorator(withCenteredStory);
+export default {
+  title: 'Pickers and Editors/ColorPicker/Palettes/NamedColorsPalette',
```

Comment:
Would be great if we could have constants for the top most categories for easier manipulation.